### PR TITLE
Fix coding standards

### PR DESCRIPTION
**Pull request by deepaklalwani97**
- Fixed indenation issues.
- Added nonce verification code.
- Enqueued script instead of directly including to html
- Escaped neccesary outputs.

### CODE REVIEW

File: b2w-redirection.php [hunk 1]
```diff
@@ -1,263 +1,332 @@
...
+	// unprepared sql ok.
+	$rand_col2    = $wpdb->get_results( $sql1 ); //phpcs:ignore
+
+	$blogger_url  = 'http://' . $domain_name . $rand_col2[0]->meta_value;
+	$blogger_link = '<a href = "' . esc_url( $blogger_url ) . '" target = "_blank">' . $blogger_url . '</a> ';
+	$local_url    = get_permalink( $rand_post_id );
+	$local_link   = '<a href = "' . esc_url( $local_url ) . '" target = "_blank">' . $local_url . '</a> ';
+
+	// escape safe.
+	echo '<h3><u>Test Case</u></h3><pre>Clicking this link &raquo; <b>' . $blogger_link . '</b><br/>Should redirect to &raquo; <b>' . $local_link . '</b></pre>'; //phpcs:ignore
```
Comment (chandrapatel) [inline b2w-redirection.php]: Use `printf()` instead of concatenation `.` operator.

File: b2w-redirection.php [hunk 2]
```diff
@@ -1,263 +1,332 @@
...
+	$rand_col2    = $wpdb->get_results( $sql1 ); //phpcs:ignore
+
+	$blogger_url  = 'http://' . $domain_name . $rand_col2[0]->meta_value;
+	$blogger_link = '<a href = "' . esc_url( $blogger_url ) . '" target = "_blank">' . $blogger_url . '</a> ';
+	$local_url    = get_permalink( $rand_post_id );
+	$local_link   = '<a href = "' . esc_url( $local_url ) . '" target = "_blank">' . $local_url . '</a> ';
+
+	// escape safe.
+	echo '<h3><u>Test Case</u></h3><pre>Clicking this link &raquo; <b>' . $blogger_link . '</b><br/>Should redirect to &raquo; <b>' . $local_link . '</b></pre>'; //phpcs:ignore 
+	die( '<p><b>If you are stuck, you can use our <a href="http://rtcamp.com/support/forum/blogger-to-wordpress/" target="_blank">free support forum</a> or <a href="http://rtcamp.com/contact/" target="_blank">hire us</a>.<br /><br />' );
```
Comment (chandrapatel) [inline b2w-redirection.php]: Make this string translatable and escape it using `wp_kses()`

File: b2w-redirection.php [hunk 3]
```diff
@@ -1,263 +1,332 @@
...
+
+add_action( 'wp_ajax_rt_b2wr_verify_config', 'rt_b2wr_verify_config' );
+
+/**
+ * Get Latest Feeds - Begin
+ */
+function rt_get_feeds_from_blogger_to_wp() {
+	include_once ABSPATH . WPINC . '/feed.php';
+
+	$rss = fetch_feed( 'http://rtcamp.com/tag/blogger-to-wordpress/feed' );
```
Comment (chandrapatel) [inline b2w-redirection.php]: Use https://bloggertowp.org site feed URL.

File: b2w-redirection.php [hunk 4]
```diff
@@ -1,263 +1,332 @@
...
+		echo '<div id="b2wr_notice_block" class="error"><p>Due to recent updates on blogger.com, Blogger to WordPress Redirection plugin has been rewritten. The process has also changed completely. Please refer the updated instructions here: <a class="blue_color" href="http://rtcamp.com/tutorials/blogger-to-wordpress-redirection-plugin/" target="_blank" title="Read more details about this update at Our Blog">(Read More…)</a><span><input type="button" id="hide_b2wr_notice_block" value="Hide this message!" class="button"></span></p></div>';
+	}
+
+}
+
+/**
+ * Hide Notice block
+ */
+function rt_b2wr_hide_notice_block() {
+
```
Comment (chandrapatel) [inline b2w-redirection.php]: Nonce verification is missing.

File: b2w-redirection.php [hunk 5]
```diff
@@ -1,39 +1,44 @@
...
+ * Description: This plugin is useful for setting up 1-to-1 mapping between Blogger.com blog posts and WordPress blog posts. This works nicely for blogs with old subdomain address (e.g. xyz.blogspot.com) which are moved to new custom domain (e.g. xyz.com)
+ * Version: 2.2.5
+ * Author: rtCamp
+ * Author URI: https://rtcamp.com/
+ * Requires at least: 3.2
+ * Tested up to: 5.1
+ * Text Domain: blogger-to-wordpress
  *
- * @package BloggerToWordpress
+ * @package blogger-to-wordpress
```
Comment (gagan0123) [inline b2w-redirection.php]: Should have been Blogger_To_WordPress, from what I remember about standards

**Pull request merged**